frontend/src/components/ToDoLists.js

```diff
@@ -11,21 +14,48 @@ export const ToDoLists = ({ lists, selectedList, setSelectedList }) => {
             return list.error
         } else {
             console.log(list)
-            setSelectedList(list) 
+            setSelectedList(list)
         }
     }
+    
+    // need to abstract away this function...
+    const handleAddItem = async (event) => {
+        if (!newListName) {
+            setErrorMessageState('Input Required');
+            return;
+        }
+        if (event.charCode === 13) {
+            const res = await addList(newListName);
+            if (res.error) {
+                setErrorMessageState(res.error);
+            } else {
+                setErrorMessageState('');
+                setMessageState('Post Successful');
```

Comment:
'Post sucessful' doesn't make sense to a user. What does Post mean to a user?

Reply:
"List Successfully Added"